### DISCUSSION TITLE
8.0.0

### DISCUSSION THREAD

**Discussion by eXorus** [php-mime-mail-parser/php-mime-mail-parser#404] [General]
Add support for PHP 8.2

<hr /><em>This discussion was created from the release <a href='https://github.com/php-mime-mail-parser/php-mime-mail-parser/releases/tag/8.0.0'>8.0.0</a>.</em>

**Comment by ebardoul** [comment]
It seems to work !

But I had to modify 2 lines in Parser.php to have it working well when calling $parser->getMessageBody( "htmlEmbedded" ) :

Line 404 :
Original : `$body = empty($inline_parts) ? '' : $inline_parts[0];`
Modified : `$body = empty($inline_parts) ? '' : $inline_parts;`

Line 424 :
Original : `return $body;`
Modified : `return $body[0];`

With original lines it return for me, only the first part from certains messages.
With the modified version, it seems to return a correct html content with all embedded.

**Reply by eXorus** [reply]
Thanks, but doesn't look good on my side:
https://github.com/php-mime-mail-parser/php-mime-mail-parser/pull/406

There are 41 errors with PHP 8.0, PHP 8.1 and PHP 8.2.
